remote.go

```diff
@@ -1030,6 +1020,10 @@ func (elem *remoteWE) toMap() map[string]string {
 	}
 }
 
+func (elem *remoteWE) MarshalJSON() ([]byte, error) {
+	return json.Marshal(elem.toMap())
```

Comment:
Maybe remove the toMap method, since it is only used here, and inline the map initialization?

(I should have caught this in #49).